src/components/bars/leftBar/Leftbar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Drawer from '@mui/material/Drawer';
+import {leftbarStyle} from "./Leftbarstyle";
+
+function Leftbar() {
+  const classes = leftbarStyle();
+  return (
+    <div>
```

Comment:
I feel this div is not needed.

Reply:
yes,It's not a harm, it will be fine without div as Box is wrapping all content